Docs/ProjectSpec.md

```diff
@@ -415,6 +421,7 @@ targets:
       - framework: path/to/framework.framework
       - carthage: Result
         findFrameworks: true
+        static: false
```

Comment:
Since this defaults to `false`, do we want to instead show the customization? Same for `findFrameworks`.

Reply:
This is an example, so we can change to `static: true` for intelligible.